File: mobile/src/main/java/com/alexstyl/specialdates/events/namedays/calendar/resource/NamedayJSONResourceLoader.java
package com.alexstyl.specialdates.events.namedays.calendar.resource;

import com.alexstyl.specialdates.events.namedays.NamedayLocale;

import org.json.JSONException;
import org.json.JSONObject;

interface NamedayJSONResourceLoader {
    JSONObject loadJSON(NamedayLocale locale) throws JSONException;
}

File: android_mobile/src/main/java/com/alexstyl/specialdates/events/namedays/calendar/resource/NamedayJSONResourceLoader.kt
package com.alexstyl.specialdates.events.namedays.calendar.resource

import com.alexstyl.specialdates.events.namedays.NamedayLocale

import org.json.JSONException
import org.json.JSONObject

internal interface NamedayJSONResourceLoader {
    @Throws(JSONException::class)
    fun loadJSON(locale: NamedayLocale): JSONObject
}
